packages/oidf-client/package.json

```diff
@@ -1,5 +1,6 @@
 {
 	"name": "@sphereon/ssi-sdk.oidf-client",
+	"type": "module",
```

Comment:
@jcmelati 
setting type to module specifies the type of modules that the project will use. and this is the only one with this option. and from my experience is most likely never the solution to the problem